lsof.yaml

```
@@ -0,0 +1,70 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/lsof/APKBUILD
+package:
+  name: lsof
+  version: 4.99.3
+  epoch: 0
+  description: LiSt Open Files
+  copyright:
+    - license: custom
```

```suggestion
    - license: lsof
```
ref: https://spdx.org/licenses/lsof.html